tools/transformer/__tests__/transformations.test.js

```diff
@@ -39,17 +39,6 @@ test("Transform AllOf model", () => {
   ).toMatchInlineSnapshot(cases.ChildAllOf);
 });
 
-test("Fail Transform AllOf with duplicates", () => {
```

Comment:
We no longer need to present that to the user. This is happening for each transformation we have.